test/e2e/flowaggregator_test.go

```diff
@@ -229,6 +288,10 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv6.String(), podBIPs.ipv6.String(), isIPv6, true, false, true, false)
 		}
 	})
+	// Only true when testing secure connection for ClickHouse DB
+	if intraNodeFlowsOnly {
+		return
+	}
```

Comment:
what does that have to do with using TLS to connect to CH?

Reply:
Deleted. I was thinking to reuse the code here and only run the first flow test to see if we can have result in clickhouse when using http or https.
I've updated this file. Thanks.